jokes.py

```diff
@@ -1,11 +1,26 @@
 import time
 from reddit import get_newest
-from twitter import post_tweet
+from twitter import post_tweet, post_thread
 
 
 def post():
     for tweet in get_newest():
-        post_tweet(tweet)
+        # One-tweet joke
+        if len(tweet) <= 280:
+            post_tweet(tweet)
+        # Threaded (multiple-tweet) joke
+        else:
+            tweets = []
+            tweet_words = tweet.split()
+            t = ""
+            for word in tweet_words:
+                # we check against 279 because we add a space
+                if (len(t) + len(word)) <= 279:
+                    t += word + " "
+                else:
+                    tweets.append(t)
+                    t = word
+            post_thread(tweets)
```

Comment:
While this also works, it would be better to split the tweet based new sentence (checking for a period) and not necessarily checking when 280 characters are completed. Sorry for not mentioning this in the issue description.

Reply:
Commit `771cf16` changes this situation. The new procedure is basically the following:

- If the joke is longer than 280 words, split it by sentences.
- If a given sentence is less than 280, tweet it as a whole and move on to the next.
- If a given sentence is still longer than 280 words, then split it by words and tweet using the previous procedure (add words until 280 characters are completed). 

What do you think?